frontend/src/components/common/SchedulingProgressBar.tsx

```diff
@@ -13,18 +13,20 @@ const SchedulingProgressBar = ({
   }));
 
   return (
-    <Flex flexDir="column" width="100%">
-      <Steps
-        activeStep={activeStep}
-        responsive={false}
-        colorScheme="raddish"
-        size="sm"
-      >
-        {steps.map(({ label }) => (
-          <Step key={label} colorScheme="raddish" />
-        ))}
-      </Steps>
-    </Flex>
+    <Container px="32px" py="35px">
```

Comment:
Can we remove this container and keep it as it was before so that the progress bar in other screens is not affected. We can style your screen the same way Anne did it for her `Volunteer Information` screen

Reply:
+1 I also styled my page the way Anne did